http/src/request/channel/message/create_message.rs

```diff
@@ -95,6 +119,20 @@ impl<'a> CreateMessage<'a> {
         Ok(self)
     }
 
+    /// Set the embed of the message.
+    ///
+    /// Embed total character length must not exceed 6000 characters.
```

Comment:
I think that if we note this here we should also maybe also note the other limitations, such as the field length and title length.